.github/workflows/build_and_test_midstream_image.yml

```diff
@@ -29,11 +29,9 @@ jobs:
       - name: "Build the midstream image inside Dockerfiles"
         run: |
           podman --version
-          cd Dockerfiles/midstream/
-          podman build -t midstream_image:latest -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0
+          podman build -t midstream_image:latest -f Dockerfiles/midstream/Dockerfile --no-cache .
```

Comment:
What is a benefit of using `--no-cache` in CI?
It was not used before and there is nothing cached in  github actions

Reply:
True. I will change it.